app/controllers/api/v4/patient_controller.rb

```diff
@@ -32,7 +32,7 @@ def activate
     head :ok
   end
 
-  def authenticate
+  def activate
```

Comment:
If we want to be consistent with the v3 naming, should we call this `login`?

Reply:
This should be `login`, and `request_otp` should be `activate` to be consistent with v4 user APIs

Reply:
That sounds right to me.

@timcheadle since that change would cause the endpoints to change on Automation Squared again, I'm going to hold off on making it. When you get in, if you agree, can you make the change and notify A2?